plugins/performance-lab/includes/site-health/object-cache/hooks.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Hook callbacks used for Site Health Info.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Adds Object Cache module to Site Health Info.
+ *
+ * @param array{object_cache: array{label: string,description: string,fields: array<string, array{label: string,value: string}>}} $info Site Health Info.
+ *
+ * @return array{object_cache: array{label: string,description: string,fields: array<string, array{label: string,value: string}>}} Amended Info.
+ * @since n.e.x.t
```

Comment:
The typing can be made a bit more generic.

```suggestion
 * @since n.e.x.t 
 *
 * @param array<string, array{label: string, description: string, fields: array<string, array{label: string, value: string}>}> $info Site Health Info.
 * @return array<string, array{label: string, description: string, fields: array<string, array{label: string, value: string}>}> Amended Info.
 * 
```